src/wifi/wifi_card.py

```diff
@@ -0,0 +1,10 @@
+class WifiCard:
+    interface_name: str
+    location: [float, float]
```

Comment:
location: [float, float] reused multiple places, perhaps an abstraction is in order.

Reply:
Point?